django_prometheus/middleware.py

```diff
@@ -141,7 +145,10 @@ def process_response(self, request, response):
         return response
 
     def process_exception(self, request, exception):
-        name = request.resolver_match.view_name or '<unnamed view>'
+        if hasattr(request, 'resolver_match'):
```

Comment:
Please add a comment at l82 indicating that this export is disabled on django < 1.x (I think 1.8)